devtools/conda-envs/test_env.yaml

```diff
@@ -10,6 +10,7 @@ dependencies:
   - numpy
   - h5py
   - e3nn
+  - dgllife
```

Comment:
I have added `dgllife` hopefully that fixes the CI.

Reply:
and DGL, hopefully  ABI stuff doesn't make our life hell

Reply:
@kaminow great work! Let me just edit the branch protection rules to remove the python 3.8 stuff.

Reply:
oh beautiful, I was wondering how to get those to go away lol